modules/ebi-metagenomics/mgnifypipelinestoolkit/rheachebiannotation/main.nf

```diff
@@ -0,0 +1,59 @@
+process MGNIFYPIPELINESTOOLKIT_RHEACHEBIANNOTATION {
```

Comment:
@chrisAta look at those name s :fear: 
All other toolkit modules are named without **MGNIFYPIPELINESTOOLKIT**

Reply:
oof! does it need the whole name to it matches the parent directory? if not if we really want to have a prefix for these modules we could do
```suggestion
process MPT_RHEACHEBIANNOTATION {
```
instead?

Reply:
I like MPT. I'll rename it.

Thoughts? React to this message with: 👍 or 👎  to vote

@KateSakharova @Ge94 @chrisAta @Vkale1 @ochkalova

Reply:
I also think too much shorting name will cause confusion. Having it like Sonya suggested (mg_toolkit/mgtoolkit) or initial name seems more clear to me

Reply:
We already use mg-tookit for the cli tool for our API https://pypi.org/project/mg-toolkit/, and there is another tool with that name 
mgtoolkit -> https://mgtoolkit.readthedocs.io/en/latest/installation.html is already "taken"... I want to avoid the confusion.

I think MgnifyPipelinesToolkit (MPT) is the best... or just keeping the long name (which is the convention). Let's see if we get a few more thoughts on Slack.

Reply:
I will vote for long name then :)

Reply:
THE PEOPLE HAVE SPOKEN